pkgs/applications/radio/qdmr/default.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  installShellFiles,
+  writeText,
+  cmake,
+  libxslt,
+  docbook_xsl_ns,
+  wrapQtAppsHook,
+  libusb1,
+  libyamlcpp,
+  qtlocation,
+  qtserialport,
+  qttools,
+  qtbase,
+}:
```

Comment:
We don't format things with alejandra. Use either nixfmt or nixpkgs-fmt.

Reply:
Aha, what makes you say that these two are okay, but alejandra isn't? Apparently I've missed the memo about allowed and forbidden formatters in nixpkgs.

Reply:
@SuperSandro2000 This has not been decided yet, but the actively-discussed [RFC 101](https://github.com/NixOS/rfcs/pull/101) is about this. Contrary to what you're saying though, the shepherd team (which I'm part of) is tending towards alejandra, not away from it. So until the RFC is over, please don't tell people what formatter they should use.

Reply:
If we are actively using two contradicting formatters it will be a complete mess.

Also the rfc is now almost open for almost 20 months and lost itself in endless, unproductive discussion also known as bikesheeding.

Reply:
It literally just had a meeting today -- there is momentum.

Reply:
Since there currently is no specified formatting that has to be used I will leave it with the current formatting.

Reply:
We haven't standardised on a single formatting program, but actually _is_ a specified formatting in this case, in the [Nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#sec-syntax):

> - Function formal arguments are written as:
> 
>   ```nix
>   { arg1, arg2, arg3 }:
>   ```
> 
>   but if they don't fit on one line they're written as:
> 
>   ```nix
>   { arg1, arg2, arg3
>   , arg4, ...
>   , # Some comment...
>     argN
>   }:
>   ```